app/Filament/Resources/RepositoriesResource.php

```diff
@@ -32,7 +24,7 @@ public static function form(Form $form): Form
             ->schema([
                 Card::make([
                     TextInput::make('name')->disabled(),
-                    Checkbox::make('fork')->disabled(),
+                    TextInput::make('branch_limit')->default(5)->disabled(),
```

Comment:
why to remove `fork`?
and why `branch_limit` is disabled?

Reply:
Yes by mistake that fork was removed and is essential for the repo structure. The branch limit is disabled because it prevents users from editing it.